### DISCUSSION TITLE
Suricata Network metadata and Strelka

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#2672] [Unsupported Versions] [answered]
Does Strelka function if you selected Suricata as the network metadata instead of zeek or does it only work if Zeek is selected?

**Comment by TOoSmOotH** [comment, marked as answer]
Currently file extraction is not set up on Suricata when used as the meta data source. #2678 is in to address this issue. Thanks for reporting it.
